django/contrib/staticfiles/management/commands/collectstatic.py

```diff
@@ -177,18 +177,21 @@ def handle(self, **options):
             destination_path = None
             message.append('.\n\n')
 
-        if self.clear:
-            message.append('This will DELETE ALL FILES in this location!\n')
-        else:
-            message.append('This will overwrite existing files!\n')
-
-        message.append(
-            'Are you sure you want to do this?\n\n'
-            "Type 'yes' to continue, or 'no' to cancel: "
-        )
+        destination_exists = (destination_path and
```

Comment:
I think the prompt should still happen if `destination_path` is None since we don't check existence in that case. Only skip the prompt if we're certain the destination doesn't exist.